### DISCUSSION TITLE
Educator having trouble applying for GitHub Education benefits

### DISCUSSION THREAD

**Discussion by LudoLogical** [community/community#147444] [GitHub Education] [answered]
### Hi everyone,

I'm the Head of Software Engineering Education at CodeDay, a/k/a Student Research and Development (SRND), an educational 501(c)3 non-profit organization listed on GitHub as CodeDay by SRND. Although we are headquartered in Seattle, WA, we do not have a physical "campus," our educators come from all over, and we only do distance learning. We also do not issue employee IDs in most cases. Crucially, this means that I - and almost all of my colleagues - do not have any _physical_ evidence of our employment. In lieu of that, I submitted photos of my computer screen displaying (A) the first part of my offer letter (the whole thing won't fit on one monitor), (B) the most recent pay stub I received (as reported by our payroll processing service), and (C) a PDF version of that pay stub. GitHub responded to the request almost immediately with the following issues:

1. The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.
2. Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.
3. Your document does not appear to indicate you as a faculty member. If you meant to apply as a student, please cancel your application and reapply [here](https://education.github.com/discount_requests/student_application).

I do not believe that any of these are issues. Regarding (1), the names "CodeDay" and "Student Research and Development" both appear in all three of my documents. Regarding (2), all three documents prominently display my legal first and last name exactly as they appear in the billing information section of my GitHub account. Regarding (3), the offer letter specifically states my current job title in its introductory paragraph.

Please let me know what, if anything, I should adjust about the evidence I have supplied and who, if anyone, I should contact regarding this. Thank you for your consideration!

**Comment by samus-aran** [comment, marked as answer]
Hi @LudoLogical 👋 

I suspect that the resolution of the image isn't high enough to see the text in order to analyse it. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. Perhaps only take an image of one piece of evidence. 

Let us know how you get on.

**Comment by LudoLogical** [comment]
Hi @samus-aran,

Thanks for the suggestion; that seems to have done the trick! I was surprised to discover that the inclusion of the signature block on the offer letter was not required.
